### PR TITLE
Feat : Alerts & Utilities

### PR DESCRIPTION
**Pull request by devananth**
=> Added Alert Component 

         Simple Alerts
         Alert With Icons
         Dismiss Alerts 
   
=> Added some utility files and structured the files

Alerts

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (VedanthB) [inline .vscode/settings.json:1]: you can add this folder in .gitignore file , so they don't get committed, 
these are just your vs code settings.

**Review by VedanthB — APPROVED** [review]
looks good to go 🚀

**Review by ashishnnnnn — COMMENTED** [review]
Preview seems fine and code is also good :+1:

File: .vscode/settings.json (hunk 1 above)
Comment (devananth) [reply .vscode/settings.json:1]: we look into it ,  Thanks

**Pull request merged**